sdk/synapse/Azure.Analytics.Synapse.Monitoring/tests/samples/Sample1_PipelineMonitoring.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Azure.Analytics.Synapse.Monitoring;
+using Azure.Analytics.Synapse.Monitoring.Models;
+using Azure.Identity;
+using NUnit.Framework;
+
+namespace Azure.Analytics.Synapse.Samples
+{
+    /// <summary>
+    /// This sample demonstrates how to monitor pipeline runs using synchronous methods of <see cref="MonitoringClient"/>.
+    /// </summary>
+    public partial class PipelineMonitoring
+    {
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/17455")]
+        [Test]
+        public void MonitorPipelineRuns()
+        {
+            string endpoint = TestEnvironment.EndpointUrl;
+            MonitoringClient client = new MonitoringClient(new Uri(endpoint), new DefaultAzureCredential());
+
+            SparkJobListViewResponse sparkJobList = client.GetSparkJobList();
+            foreach (var sparkJob in sparkJobList.SparkJobs)
+            {
+                if (sparkJob.State == "Running")
```

Comment:
This is one of those things that should definitely be an enum/extensible enum if you're keeping notes for future work.